src/dom/dom.js

```diff
@@ -348,6 +348,11 @@ function addElement(elt, pInst, media) {
  * createDiv('this is some text');
  * </code></div>
  */
+p5.prototype.createDiv = function(html) {
+  let elt = document.createElement('div');
+  if (typeof html !== undefined) elt.innerHTML = html;
```

Comment:
Use a string here. 
`typeof html !== 'undefined'`
`typeof something` gives the type of something but as a string.
This will remove the test failure

Reply:
yes, that would solve this. but still the tests fail. I m looking into it.

Reply:
found the issue, this tests are not updated. there is another commit which enhances the tests. Need to find a middle out